docs/content/policy-language.md

```diff
@@ -1783,6 +1783,48 @@ them to avoid naming conflicts, e.g., `org.example.special_func`.
 See the [Policy Reference](../policy-reference#built-in-functions) document for
 details on each built-in function.
 
+### Errors
+
+By default, built-in function calls that encounter runtime errors evaluate to
+undefined (which can usually be treated as `false`) and do not halt policy
+evaluation. This ensures that built-in functions can be called with invalid
+inputs without causing the entire policy to stop evaluating.
+
+In most cases, policies do not have to implement any kind of error handling
+logic. If error handling is required, the built-in function call can be negated
+to test for undefined. For example:
+
+```live:eg/errors:module,merge_down
```

Comment:
I think the `/` trips up live blocks, https://github.com/open-policy-agent/opa/blob/27c8d75d3528dc8a04e716d5455f97ee7fb537bc/docs/website/scripts/live-blocks/src/constants.js#L30